### ISSUE TITLE
AI Sight Bugs

### ISSUE THREAD

**Issue by Afya212** [Skyrat-SS13/Skyrat13#2013]
## Reproduction:
Jump to crew members and eventually you'll randomly stop requiring cameras to see.
You can also see where someone is talking if outside your cameras via runchat.

## Possible Screenshots of the Issue in question:
![image](https://user-images.githubusercontent.com/54920261/82162725-af209d80-9874-11ea-83bd-310c2f3a2e82.png)
![image](https://user-images.githubusercontent.com/54920261/82162744-cfe8f300-9874-11ea-863a-5ce0cc3dbf8d.png)

**Comment by SpaceVampire** [comment]
That first bug is fuckin ancient. Its been an issue with the AI for as long as I have been playing.

**Comment by IronEleven** [comment]
Yeah, best Cit ever managed with it was warning the AI to relog whenever it tried to interact with something it wasn't supposed to see.
